auditLogMover/tsconfig.json

```diff
@@ -1,8 +1,8 @@
 {
   "compilerOptions": {
     "target": "es2019", // Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019' or 'ESNEXT'.
-    "module": "commonjs", // Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', or 'ESNext'.
     "strict": true, // // Enable all strict type-checking options.
+    "moduleResolution": "node",
```

Comment:
Removing "module" per the recommendation [here](https://github.com/AnomalyInnovations/serverless-bundle/issues/124#issuecomment-716603300)

Serverless bundle doesn't support CommonJs
https://github.com/AnomalyInnovations/serverless-bundle#module-and-targets

Reply:
Does this need to be propagated to our other packages?

Reply:
Our other packages as in `persistence`, `interface`, etc...? 

In my testing no changes needed to be made to the other packages for serverless deploy to work correctly. My IDE also behave as expected. I also tested linking the deployment package to local version of the routing package. I made code changes and the changes was reflected back and ran successfully.

Reply:
Awesome thanks